sys/kern/sbrk.c

```diff
@@ -22,8 +22,8 @@ void sbrk_attach(proc_t *p) {
   /* Initially allocate one page for brk segment. */
   vaddr_t addr = SBRK_START;
   vm_object_t *obj = vm_object_alloc(VM_ANONYMOUS);
-  vm_segment_t *seg =
-    vm_segment_alloc(obj, addr, addr + PAGESIZE, VM_PROT_READ | VM_PROT_WRITE);
+  vm_segment_t *seg = vm_segment_alloc(obj, addr, addr + PAGESIZE,
+                                       VM_PROT_READ | VM_PROT_WRITE, 0);
```

Comment:
`vm_flags_t` is a enum. `VM_FILE` is easier to read than `0`.

Reply:
Okay, I will change it